.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -673,7 +673,7 @@ protected double GetDigitalValue(string digitStr, double power)
             for (var i = 0; i < digitStr.Length; i++)
             {
                 var ch = digitStr[i];
-                if (!isFrac && (ch == Config.NonDecimalSeparatorChar || ch == ' '))
+                if (!isFrac && (ch == Config.NonDecimalSeparatorChar || ch == ' ' || ch == ' '))

```

Comment:
Same constant comment applies here. Otherwise any reader of the code will be confused with the "duplicate" check.